docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -88,8 +89,23 @@ kubectl apply -f src/test/system_tests/ceph_s3_tests/test_ceph_s3_job.yml
 kubectl logs job/noobaa-tests-s3 -f
 ```
 
+#### Skipped tests
+
 We run all the tests except the tests that appear in the lists `src/test/system_tests/ceph_s3_tests/s3-tests-lists` if you would like to add or remove a test you can edit those files (and then repeat the steps starting from 'Build Core And Tester Images (Noobaa-Core)' above).
 
+Some tests are marked to be skipped in the tests code. usually because the enviorment doesn't support certain criteria. those tests will appear with a `[WARN]` tag and will be marked as "Test Skipped", for example:
+`[WARN] CONSOLE:: Test skipped: s3tests_boto3/functional/test_s3.py::test_lifecycle_transition`
+
+In the test code the function:
+`pytest.skip("")` will mark them to be skipped.
+
+#### Boto3 tests
+Most of the tests have both Boto3 and Boto2 versions. Tests written in Boto2 are in the s3tests directory. Tests written in Boto3 are located in the s3test_boto3 directory.
+
+in order to only run the boto3 version, tests in the boto directory run with `-- -m 'not fails_on_aws`:
+
+`S3TEST_CONF=your.conf tox -- -m 'not fails_on_aws' s3tests_boto3/functional::test_name`
```

Comment:
I would just mention that this part was taken from the Ceph S3-Tests readme (and add the link to that point), in case the doc would not be updated and it might be confusing.